#!/usr/bin/python3
def raise_exception_msg(message=""):
    """ Raise a NameError with the supplied message
    """
    raise NameError(message)
